src/Integration/Doctrine/DoctrineORMMiddleware.php

```diff
@@ -91,7 +90,7 @@ private function ping(Connection $con): bool
             $con->executeQuery($con->getDatabasePlatform()->getDummySelectSQL());
 
             return true;
-        } catch (Exception | DBALException $e) { // @phpstan-ignore-line
```

Comment:
Maybe if you rollback this and add `reportUnmatchedIgnoredErrors: false` to `phpstan.neon.dist`, it will work pass.